lib/mustache/generator.rb

```diff
@@ -118,6 +120,17 @@ def t.tokens(src=@source)
             p.compile(src)
           end
           t.render(ctx.dup)
+        compiled
+      end
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [`end` at 124, 6 is not aligned with `if` at 110, 22.](https://www.codacy.com/app/rokusu/mustache/file/2003859940/issues/source?bid=2903613&fileBranchId=3121922#l124)